### PR TITLE
Rename "Contractors" column on project list to "Workgroup/Contractors"

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/14249

I renamed the `contractors` column on the project list to `workgroup_contractors`, as well as renaming the `contractor` column on the `moped_proj_work_activity` table to be be `workgroup_contractor`. 

I also snuck in making the task orders in the project list view to be distinct, so they are not repeated in the project list table.

## Testing
**URL to test:** 

Test locally

**Steps to test:**

spin up the editor. The column previously titled "Contractors" on the project list is now titled "Workgroup/Contractors"

Go to a project and test the renaming of the contractor column by:
1. Navigating to the funding tab. Add a new work activity, and populate the workgroup/contractor field by either selecting a workgroup from the dropdown or typing in a new contractor name. Save. 
2. Confirm the activity appears on the funding tab, with expected workgroup contractor.
3. Edit an activity, changing the name of the contractor, confirm the update mutation is also working. 
4. Duplicate some task orders (different work activities, same task order) and check that the task orders only appear once in the Projects list. 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/migrations/1699978012000_change_contractor_column_name/down.sql
Comment (mddilley) [inline moped-database/migrations/1699978012000_change_contractor_column_name/down.sql:1]: I ran into a small wrinkle running this down migration - `column mpwa.contractor does not exist`

**Review by mddilley — COMMENTED** [review]
I tested out the steps, and this works great! 🙏 🙌 I ran into a small problem with a down migration and left a note.

File: moped-database/migrations/1699978012000_change_contractor_column_name/down.sql
Comment (chiaberry) [reply moped-database/migrations/1699978012000_change_contractor_column_name/down.sql:1]: thanks, i swapped the order of the migrations and missed updating this part. I will fix now!

**Comment by chiaberry** [comment]
also found an error when applying the seeds, i will fix that today too

**Review by johnclary — APPROVED** [review]
yeesh—what a brutal amount of code change to rename a column 😞 

thank you for working through all these changes. everything looks good to me! i will make a note to update the AGOL dataset columns when we release 🙌

**Comment by chiaberry** [comment]
I am going to have to combine the separate migration files i have here into one, I dont think they can be done independent of each other. will update that here shortly

**Comment by mddilley** [comment]
Cool @chiaberry - thanks for taking a look. I agree that the view dependencies can get so gnarly! I'm due for one of these for sure. 😄

**Review by mddilley — APPROVED** [review]
Awesome!!! Thanks for deduping the task orders to in the project list view too. No more key warnings! 🙌 🚀

**Pull request merged**